from ...common.base.exceptions import BaseAppException


class UserAlreadyExistsException(BaseAppException):
    """User already exists exception"""
